releases/nightly-build/compose-files/docker-compose-nexus-redis.yml

```
@@ -0,0 +1,574 @@
+# /*******************************************************************************
+#  * Copyright 2020 Redis Labs
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * Andre Srinivasan
+#  * added: April, 2020
+#  *******************************************************************************/
+
+version: '3.4'
+
+# all common shared environment variables defined here:
+x-common-env-variables: &common-variables
+  Registry_Host: edgex-core-consul
+  Clients_CoreData_Host: edgex-core-data
+  Clients_Logging_Host: edgex-support-logging
+  Clients_Notifications_Host: edgex-support-notifications
+  Clients_Metadata_Host: edgex-core-metadata
+  Clients_Command_Host: edgex-core-command
+  Clients_Scheduler_Host: edgex-support-scheduler
+  Logging_EnableRemote: "true"
+  Databases_Primary_Type: redisdb
+  Databases_Primary_Host: edgex-redis
+  Databases_Primary_Port: 6379
+  SecretStore_Host: edgex-vault
+  SecretStore_ServerName: edgex-vault
+  SecretStore_RootCaCertPath: /tmp/edgex/secrets/ca/ca.pem
+
+x-redis5-env-variables: &redis5-variables
+  REDIS5_PASSWORD_PATHNAME: /dbsecrets/redis5-password
+
+volumes:
+  db-data:
+  db-secrets:
+  log-data:
+  consul-config:
+  consul-data:
+  consul-scripts:
+  vault-init:
+  vault-config:
+  vault-file:
+  vault-logs:
+  # non-shared volumes
+  portainer_data:
+  secrets-setup-cache:
+
+services:
+  volume:
+    image: nexus3.edgexfoundry.org:10004/docker-edgex-volume:master
```

Trying to get rid of edgex-volume.

```suggestion
    image: ubuntu:latest
    entrypoint: [ "tail", "-f", "/dev/null" ]

```